prom-bundle/alerts/upness.rules

```diff
@@ -0,0 +1,20 @@
+ALERT NodeDown
+  IF up{job="node"} == 0
+  FOR 5m
```

Comment:
This is where we run into the first issues with generalizability of examples I guess. In environments that we are used to, you shouldn't page on a single down hosts anymore and only about about service-level symptoms. But I guess many people are not yet at that point. Still, happy to change it as it is the right way forward...